benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -814,7 +825,7 @@
     "ID": "draftToIncompleteApplication",
     "Name": "Draft to incomplete appeal",
     "Description": "Convert draft case to incomplete application received",
-    "DisplayOrder": 69,
+    "DisplayOrder": 71,
```

Comment:
The order 69 should be changed to 70, this would affect the display
All previous display with order 69 should be made 70